gp-includes/routes/translation.php

```diff
@@ -28,7 +28,11 @@ public function import_translations_get( $project_path, $locale_slug, $translati
 			return $this->die_with_404();
 		}
 
-		if ( $this->cannot_and_redirect( 'approve', 'translation-set', $translation_set->id ) ) {
+		$can_import = $this->can( 'import', 'translation-set', $translation_set->id );
+		$can_only_import_waiting = ! $can_import && $this->can( 'import-waiting', 'translation-set', $translation_set->id );
+
+		if ( ! $can_import && ! $can_only_import_waiting ) {
```

Comment:
The same logic should be added to https://github.com/GlotPress/GlotPress-WP/blob/873ff71b83b0d7abf7addda12b7ee8b14dcef5f5/gp-templates/translations.php#L237-L239 to avoid displaying a link which a user can't access.